docs/output.md

```diff
@@ -55,6 +55,15 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 <details markdown="1">
 <summary>Output files</summary>
 
+- `transformation/`
+  - A CSV-formatted file for reporting to IRIDA Next the populated column for each sample: `transformation.csv`
```

Comment:
"the populated column" should be changed to something like the "source-categorized column" or similar.